src/completers/git.zsh

```diff
@@ -726,7 +789,7 @@ _fzf_complete_git-commits-not-in-head() {
     local fzf_options=$1
     shift
 
-    local rev_list_all=$(git rev-list --all --oneline 2> /dev/null)
+    local rev_list_all=$(git rev-list --branches --tags --remotes --oneline 2> /dev/null)
```

Comment:
In the case of using `--all`, `refs/stash`, which is the latest stash, is also shown.